telemetry/convert.py

```diff
@@ -36,6 +37,8 @@ class Converter:
     """A class for converting incoming payloads to a more compact form"""
     VERSION_UNCONVERTED = 1
     VERSION_CONVERTED = 2
+    # the unified FHR/Telemetry ping has a top-level version of 2 and payload version of 4
+    VERSION_UNIFIED = 2
```

Comment:
I think you should use the payload version (4) here, and use `ver` vs. `version` for top-level detection of unified pings. Otherwise a "2" in a converted ping is ambiguous.